pkg/util/utils.go

```diff
@@ -149,9 +149,6 @@ func SummarizeTestResults(
 		}
 
 		s := sorted[k]
-		s.Successes = passedCount
-		s.Failures = failedCount
-		s.TestPassPercentage = Percent(passedCount, failedCount)
```

Comment:
i'm surprised you don't think this is of interest (no, it's not in the html render today, but it could be added).

it provides a nuance that "passing job percentage" does not.  (some platforms may be consistently failing because of a single bad test. other platforms may be consistently failing because of hundreds of test failures)